core/src/main/scala/kafka/server/DelegationTokenManagerZk.scala

```diff
@@ -192,6 +193,10 @@ class DelegationTokenManagerZk(config: KafkaConfig,
     }
   }
 
+  private def sum(now: Long, duration: Long): Long = {
```

Comment:
the zk is dropping and we don't add tests for zk, so could you please revert those changes?

Reply:
Done